src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestMessagesProcessor.cs

```diff
@@ -24,8 +28,20 @@ public WasmTestMessagesProcessor(string xmlResultsFilePath, string stdoutFilePat
             this._logger = logger;
         }
 
-        public void Invoke(string line)
+        public void Invoke(string message)
         {
+            WasmLogMessage? logMessage = null;
+            string line;
+            try
+            {
+                logMessage = JsonSerializer.Deserialize<WasmLogMessage>(message);
```

Comment:
@radical can we check whether the `message` starts with `{` before trying to json-deserialize it to avoid throwing a lot of exceptions?